### DISCUSSION TITLE
v1.1

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#2798] [General]
**The installers are located in the "Assets" section at the end of these release notes!**

[![Github Release](https://img.shields.io/github/downloads/Stellarium/stellarium/v1.1/total.svg)](https://github.com/Stellarium/stellarium/releases/v1.1)

## Release notes
The major changes of this version:
- Added support OpenGL 3.3 Core profile
- Changed core: switch to use CalcMySky v0.2.1
- Changed GUI: allow user CSS
- Fixed compiling with Qt 6.4
- Fixed Telescope Control plugin
- Removed Dakota & Ojibwe sky cultures: this is on request of the original contributors, and following a misunderstanding about licensing.

## What's Changed
Full list of changes:
- Added support OpenGL 3.3 Core profile
- Added support for the versioned variant of ShowMySky library
- Added ability to show only special nomenclature points (GH: #2748)
- Added Abort slew button into Telescope Control plugin
- Added aberration to CustomObjects made from Simbad queries (GH: #2769)
- Added donation link to AppStream Metadata (GH: #2773)
- Added small tuning for DSO selection without magnitudes (GH: #2701, #2716)
- Added another vanished timezone
- Added workaround for QTBUG-105984 in macOS (GH: #2785)
- Fixed loading of glow texture in OnlineQueries plugin
- Fixed compiling with Qt 6.4: use QString instead of QByteArray (GH: #2709)
- Fixed OnlineQueries plugin button: redesigned OnlineQueries icons (GH: #2710, #2714)
- Fixed building Stellarium with -DENABLE_MEDIA=Off option
- Fixed using of OpenGL debug logger
- Fixed AppVerName variable for Windows installer (GH: #2728)
- Fixed crash in AtmosphereShowMySky constructor when OpenGL 3.3 functions pointer is null
- Fixed loading dynamic plugins (GH: #2725)
- Fixed bundling for the versioned variant of ShowMySky library on macOS
- Fixed resizing tab names in Help window when language is changed
- Fixed a missing iterator count-up(!) in SlewDialog (Telescope Control plugin)
- Fixed saving option "Use horizontal coordinates" in Oculars plugin
- Fixed saving mount properties in Telescope Control plugin (GH: #2713)
- Fixed protocol for screenshots URL
- Fixed bundling latest versions of ShowMySky library (GH: #2761, #2779)
- Fixed normals map of the Moon, increase its resolution and implement shadows (GH: #2781)
- Fixed issue in unit testing of scripting engine (GH: #2774)
- Fixed freezing Stellarium when scripts are running (GH: #2792)
- Changed core: make Date&Time dialog more usable for typing date
- Changed core: optional support XLSX files (GH: #2723)
- Changed core: avoid deprecation warnings on Qt5.14+
- Changed core: avoid a debug output
- Changed core: avoid uninitialized vectors
- Changed core: replace world map with a higher-resolution version (The map was taken from NASA Blue Marble Next Generation dataset for July 2004) (GH: #2742)
- Changed core: handle exceptions from AtmosphereRenderer when drawing
- Changed core: current atmosphere should be deleted immediately if we need to reset to fallback
- Changed core: handle the options that set environment variables before QGuiApplication is created
- Changed core: detect OpenGL ES mode dynamically rather than by conditional compilation
- Changed core: make orbits antialiased
- Changed core: simplify check for GLES
- Changed ASCOM client in Telescope Control plugin
- Changed core: switch to use CalcMySky v0.2.1
- Changed core: restore current OpenGL context if it got replaced after window resize during StelMainView::init()
- Changed core: avoid GL_INVALID_OPERATION caused by operating on zero VAO
- Changed core: move dithering mode management from StelPainter to StelCore
- Changed rules for packaging Stellarium in Windows
- Changed rules for packaging Stellarium in Linux
- Changed license info for Boorong sky culture
- Changed license info for Lokono sky culture
- Changed GUI: allow user CSS (GH: #2772, #678, #1624, #2763)
- Changed Telescope Control plugin: trivial simplifications)
- Updated scripts
- Updated list of default locations
- Updated default catalog of pulsars
- Updated GUI: set date delimiters in Date-Time dialog in one place
- Updated GUI of Telescope Control plugin
- Updated list of default locations
- Updated description of Equation of Time plugin
- Updated AppStream Metadata
- Removed outdated bookmark in Solar System Editor plugin
- Removed languages without translators
- Removed Dakota & Ojibwe sky cultures: this is on request of the original contributors, and following a misunderstanding about licensing.
- Removed the font hook for Windows

## Packages notes
package | note
------------|---------
stellarium-1.1.tar.gz | Stellarium 1.1 for UNIX/Linux (source code). <br />__MD5:__ 329a774ba484016db192a79d1c034ae4 <br />__SHA256:__ 224af67a5ff92aed3473483797e7baf41d2ba96920a2d9e4003413f9e6744fd5
stellarium-1.1-qt5-win32.exe | Stellarium 1.1 (v0.22.4) for Windows (32-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: b8428389d27a71bce8590155cbadead8 <br />__SHA256__: 23568ac56d63d0d72aa6bfc5bb89eb407b3ebb1b8e451781e37ade78422a470f
stellarium-1.1-qt5-win64.exe | Stellarium 1.1 (v0.22.4) for Windows (64-bit; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017 Community) with media support. <br /> __MD5__: 5f5da183a20de425e874e732839d733d <br />__SHA256__: d453630c70abd0036cf0c4dcc99eb235199c45220e982b9205641b00ef076d35
stellarium-1.1-qt6-win64.exe | Stellarium 1.1 (v1.22.4) for Windows (64-bit; Windows 10+) based on Qt 6.4.0 (Visual Studio 2019 Community) with media support. <br /> __MD5__: 9e0b8f0b4a6971fb182c228807e25713 <br />__SHA256__: f26a6a0b046877180ae0826e8095652d76d27ae369c5d0372ed001346034d459
Stellarium-1.1-x86_64.zip | Stellarium 1.1 (v0.22.4) for macOS (x86_64; macOS 10.15+) based on Qt 5.12.12 (AppleClang 11.0.0) with media support. <br /> __MD5__: 736cfe466bee947ea614186eb3d8b124 <br />__SHA256__: f3b6b4cbcb7322c6c5d3ff861ed3fde4d261eb03e7c8a197714874ee570a8769
Stellarium-1.1-macOS.zip | Stellarium 1.1 (v1.22.4) for macOS (universal; macOS 11.0+) based on Qt 6.4.0 (AppleClang 14.0.0) with media support. <br /> __MD5__: 1618188e48f37b35293a61898ef71507 <br />__SHA256__: 3225237eeb762bb45eea356df0b2054f2bbf1c34b1235b86ce3777a74916a2f6
Stellarium-1.1-x86_64.AppImage | Stellarium 1.1 (v0.22.4) for Linux (amd64; glibc 2.31) based on Qt 5.12.8 (GCC 9.4.0) with media support. <br /> __MD5__: cca84b1fb40dcec71bf2ac5f111aedfc <br />__SHA256__: 0202be4831885f56f0c6b17c23830b48e6a55d009eac9fd48e1f01bb61a71fb7

<hr /><em>This discussion was created from the release <a href='https://github.com/Stellarium/stellarium/releases/tag/v1.1'>v1.1</a>.</em>

**Comment by 10110111** [comment]
You forgot a major change: ShowMySky model is now supported on macOS!

**Comment by tflyer-rgb** [comment]
Hello,

Thank you for the excellent software, unfortunately the application can no longer be started after the update from 1.0 to 1.1. Uninstalling and reinstalling did not help. The log_file shows the following.
[log.txt](https://github.com/Stellarium/stellarium/files/9903026/log.txt)

Kind regards

Tom

**Reply by alex-w** [reply]
Thanks for report! This is very bad news - Qt5-based version is broken now :(

**Comment by spierrard** [comment]
Where is the 32 bits version ?  i can'- find stellarium-1.1-qt5-win64.exe and stellarium-1.1-qt5-win32.exe.
Thanks

**Reply by alex-w** [reply]
> Where is the 32 bits version ? i can'- find stellarium-1.1-qt5-win64.exe and stellarium-1.1-qt5-win32.exe.
> Thanks

Both will be available tomorrow

**Comment by tflyer-rgb** [comment]
Thanks for the quick reply, some additional information with the appimage version the same problem occurs. After updating the Windows (Windows 10) application starts but shows only a black screen.

Many greetings
Tom
[log-v1.0.txt](https://github.com/Stellarium/stellarium/files/9903420/log-v1.0.txt)
[log-v1.1.txt](https://github.com/Stellarium/stellarium/files/9903421/log-v1.1.txt)

**Comment by tflyer-rgb** [comment]
Thank you for the quick fix, both are now working normal package update and appimage

**Comment by tflyer-rgb** [comment]
I just try the new  stellarium-1.1.1-qt6-win64.exe  in angle, mesa and normal mode it always starts but I only get a black screen.
[log-v1.1.1.txt](https://github.com/Stellarium/stellarium/files/9904817/log-v1.1.1.txt)

**Reply by 10110111** [reply]
What if you pass `--opengl-compat` option?

**Reply by tflyer-rgb** [reply]
passing --opengl-compat don't change the behavior
[log.txt](https://github.com/Stellarium/stellarium/files/9909239/log.txt)

**Reply by 10110111** [reply]
Please report this as an issue. Apparently, it needs quite a bit of investigation, because this is something we don't observe.

**Reply by tflyer-rgb** [reply]
I open issue #2803

**Comment by oldherl** [comment]
The title says 1.1 but the filename is 1.1.1. This difference has caused my automated build script to fail and is confusing. What version should I call it anyway?

**Reply by DarthGandalf** [reply]
I'd say it's still 1.1

1.1.1 is used only in a single place, in URL, but even there there are 2 numbers, and one of them is 1.1

**Comment by phomer60** [comment]
I can no longer run under Mac OS 10.15.7, which was only updated in July. and is the latest available OS for my Mac.
Pop Up message is the application requires MacOS 11.0.

**Comment by jam3s007** [comment]
1.1 AMDx86_64 Appimage fails to launch on Pop-OS 22.04 (Ubuntu distro). Going back to old version.
